Frameworks/Foundation/NSCFString.mm

```diff
@@ -89,12 +93,12 @@ - (instancetype)initWithFormat:(NSString*)format locale:(id)locale arguments:(va
     CFStringRef str;
 
     if (locale == nil) {
-        str = CFStringCreateWithFormatAndArguments(nullptr, nullptr, static_cast<CFStringRef>(format), argList);
+        str = _CFStringCreateWithFormatAndArgumentsAux(nullptr, &_NSCFStringCopyDescription, nullptr, static_cast<CFStringRef>(format), argList);
```

Comment:
Foundation seems capable of linking this alright; I'll check on ARM.

---
In reply to: [71446639](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71446639) [](ancestors = 71446639)